certificate-manager/src/test/java/io/strimzi/certs/OpenSslCertManagerIT.java

```diff
@@ -78,7 +78,7 @@ public void testGenerateRootCaCertWithDays() throws Exception {
         X509Certificate x509Certificate1 = loadCertificate(cert);
         assertTrue(selfVerifies(x509Certificate1),
                 "Unexpected self-verification");
-        assertEquals(x509Certificate1.getSubjectDN(), x509Certificate1.getIssuerDN(), "Unexpected self-signedness");
```

Comment:
I assume these methods were removed?

Reply:
They are deprecated as they expose some internal class or something. So where possible, I replaced it with the alternative which should be used now.